.github/workflows/check-unlabeled-issues.yml

```
@@ -0,0 +1,22 @@
+name: 'Check unlabeled issues'
```

```suggestion
name: 'Find issues with missing labels'
```

This explains better what the Action will do, and will search for.

Also, rename the `.yml` file so it matches the new name.